pkg/morph/deploy/deploy.go

```diff
@@ -203,17 +205,13 @@ func Deploy(ctx context.Context, prm Prm) error {
 
 	prm.Logger.Info("Notary service successfully enabled for the committee")
 
-	onNotaryDepositDeficiency, err := initNotaryDepositDeficiencyHandler(ctx, prm.Logger, prm.Blockchain, prm.LocalAccount)
-	if err != nil {
-		return fmt.Errorf("construct action depositing funds to the local account's Notary balance: %w", err)
-	}
+	go autoReplenishNotaryBalance(ctx, prm.Logger, prm.Blockchain, prm.LocalAccount, chNewBlock)
```

Comment:
Looks clearer now, let's keep it in a separate commit, I'll do some testing of this code.